src/wooting-usb.h

```diff
@@ -17,16 +17,32 @@ typedef enum RGB_PARTS {
 	PART4
 } RGB_PARTS;
 
+
+typedef enum WOOTING_DEVICE_TYPE {
+	// 10 Keyless Keyboard. E.g. Wooting One
+	DEVICE_KEYBOARD_TKL = 1,
+	
+	// Full Size keyboard. E.g. Wooting Two
+	DEVICE_KEYBOARD = 2
+} WOOTING_DEVICE_TYPE;
+
 typedef struct WOOTING_USB_META {
 	bool connected;
-	bool wooting_one;
-	bool wooting_two;
-} WOOTING_USB_META;
+	const char* model;
+	uint8_t max_rows;
+	uint8_t max_columns;
+	uint8_t keycode_limit;
```

Comment:
Keycode_limit makes me think of the indexes we use for the analog SDK. Can we use something like led_index?

Reply:
Yeah, could change it to `led_id_max` or something like that for it to be more obvious exactly what it means in this case